packages/chakra-ui/README.md

```diff
@@ -0,0 +1,131 @@
+[![Build Status][build-shield]][build-url]
+[![npm][npm-shield]][npm-url]
+[![npm downloads][npm-dl-shield]][npm-dl-url]
+[![Contributors][contributors-shield]][contributors-url]
+[![Apache 2.0 License][license-shield]][license-url]
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/rjsf-team/react-jsonschema-form">
+    <img src="./logo.png" alt="Logo" width="340">
+  </a>
+
+  <h3 align="center">@rjsf/rjsf-chakra-ui</h3>
+
+  <p align="center">
+  Chakra UI theme, fields and widgets for <a href="https://github.com/rjsf-team/react-jsonschema-form/"><code>react-jsonschema-form</code></a>.
+    <br />
+    <a href="https://react-jsonschema-form.readthedocs.io/en/latest/"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://rjsf-team.github.io/react-jsonschema-form/">View Playground</a>
+    ·
+    <a href="https://github.com/rjsf-team/react-jsonschema-form/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rjsf-team/react-jsonschema-form/issues">Request Feature</a>
+  </p>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [About The Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Roadmap](#roadmap)
+- [Contributing](#contributing)
+- [Contact](#contact)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+[![@rjsf/chakra-ui Screen Shot][product-screenshot]](https://rjsf-team.github.io/@rjsf/chakra-ui)
+
+Exports `chakra-ui` theme, fields and widgets for `react-jsonschema-form`.
+
+### Built With
+
+- [react-jsonschema-form](https://github.com/rjsf-team/react-jsonschema-form/)
+- [Chakra UI](https://chakra-ui.com/)
+- [TypeScript](https://www.typescriptlang.org/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+### Prerequisites
+
+- `@chakra-ui/react >= 1.7.0`
+- `react >= 16.0.0`
+- `@rjsf/core >= 2.0.0`
+
+```bash
+yarn add @chakra-ui/react
```

Comment:
I think with the documentation here, we could try to stay as verbose as possible. and provide full installation instructions for chakra. Can also refer this link https://chakra-ui.com/docs/getting-started#installation

`yarn add @chakra-ui/react @emotion/react @emotion/styled framer-motion`

Reply:
Update: even better `yarn add @chakra-ui/react@1.7.3 @emotion/react@11.7.0 @emotion/styled@11.6.0 framer-motion@5.5.5`    though I am not certain if this will make it rigid for intending users, but being able to tie the installation version can help them with reproducing errors just in case this package isn't updated with a new breaking change version of any of the above peer dependencies.

Reply:
> I think with the documentation here, we could try to stay as verbose as possible. and provide full installation instructions for chakra. Can also refer this link https://chakra-ui.com/docs/getting-started#installation
> 
> `yarn add @chakra-ui/react @emotion/react @emotion/styled framer-motion`

I like this idea. The other seems very restrictive. I scoped it to the Major version only as per the chakra docs.